lib/elixir/test/elixir/kernel/dialyzer_test.exs

```diff
@@ -54,11 +54,11 @@ defmodule Kernel.DialyzerTest do
     {:ok, [outdir: dir, dialyzer: dialyzer]}
   end
 
-  @tag otp19: false
-  test "no warnings on valid remote calls", context do
-    copy_beam! context, Dialyzer.RemoteCall
-    assert_dialyze_no_warnings! context
-  end
+  # @tag otp19: false
```

Comment:
The dialyzer bug is fixed in `maint`, which means it comes in 19.1. At that point we would need a different tag system.

However skipping this test in OTP 18 hides other dialyzer warnings that require the fallback `?generated` to fix. I think we should always run this test but add a dialyzer attributes to the module `Dialyzer.RemoteCall` to hide specific function/warnings combinations based on the version of the dialyzer application.